helloworld-jersey/src/main/resources/service-configuration.yml

```diff
@@ -1,11 +1,11 @@
 serviceId: hellojersey
-eureka:
-    hostname: localhost
-    ipAddress: 127.0.0.1
-    port: 10016
+#eureka:
```

Comment:
Do not put comments when it is ready for merge ...

Reply:
Removed commented lines.

Reply:
Removed all commented code and config that I found.